### PR TITLE
Kuzbo 119 collection summary

### PR DESCRIPTION
**Pull request by jenow**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "jquery": "^2.1.3",
     "kuzzle-sdk": "^1.9.1",
     "materialize-css": "^0.97.6",
+    "q": "^1.4.1",
```
Comment (AnthonySendra) [inline package.json]: Could we use bluebird instead of q?

**Comment by AnthonySendra** [comment]
Dispite q instead of bluebird :+1:

File: test/unit/specs/data/actions.spec.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        if (triggerError[1]) {
+          cb({message: 'error'})
+        } else {
+          cb(null, ['collection1', 'collection2'])
+        }
+      }
+    }
+  })
+
+  it('should set an error on listIndexes', (done) => {
```
Comment (xbill82) [inline test/unit/specs/data/actions.spec.js]: Do you mean "with no arguments"?

File: test/unit/specs/data/actions.spec.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    testAction(actions.listIndexesAndCollections, [], {}, [
+      { name: 'SET_ERROR', payload: ['error'] }
+    ], done)
+  })
+
+  it('should set an error on listCollections', (done) => {
```
Comment (xbill82) [inline test/unit/specs/data/actions.spec.js]: Do you mean "with no arguments"?

File: test/unit/specs/data/actions.spec.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+            } else {
+              cb(null, {mapping: 'mapping'})
+            }
+          }
+        }
+      }
+    }
+  })
+
+  it('should set an error on getMapping', (done) => {
```
Comment (xbill82) [inline test/unit/specs/data/actions.spec.js]: Do you mean "with no arguments"?

**Comment by xbill82** [comment]
:+1: a part from q (I agree we should use Bluebird)

**Pull request merged**